src/Scene/AnimationPlayer.js

```diff
@@ -0,0 +1,231 @@
+const FRAMERATE = 60;
+const INTERVAL_TIME = 1000 / FRAMERATE;
+// if is true console.log are enabled to sniff animation'state
+const debugAnimation = false;
+
+// player statut
+const PLAYER_STATE = {
+    // player is stopped
+    STOP: 0,
```

Comment:
What's the difference between `STOP` and `PAUSE`? (there's no `resume` function)